modules/expectations/mobile_expectations.lua

```diff
@@ -0,0 +1,42 @@
+local expectations = require('expectations')
+local events = require('events')
+
+local Expectation = expectations.Expectation
+local Event = events.Event
+local SUCCESS = expectations.SUCCESS
+local FAILED = expectations.FAILED
+
+local module = {}
+local mt = { __index = { } }
+
+function mt.__index:ExpectEvent(event, name)
```

Comment:
@VVeremjova please provide descriptions for public functions. 
Use doxyegen comments style as in https://github.com/alecchen/doxygen-lua/blob/master/example/example.lua